docs/documentation/server_admin/topics/clients/oidc/con-advanced-settings.adoc

```diff
@@ -94,6 +94,39 @@ See https://datatracker.ietf.org/doc/html/draft-ietf-oauth-mtls-08#section-3[Mut
 Currently, {project_name} client adapters do not support holder-of-key token verification. {project_name} adapters treat access and refresh tokens as bearer tokens.
 ====
 
+[[_dpop-bound-tokens]]
+*OAuth 2.0 Demonstrating Proof-of-Possession at the Application Layer (DPoP)*
+
+DPoP binds an access token and a refresh token together with the public part of a client's key pair. This binding prevents an attacker from using stolen tokens.
+
+This type of token is a holder-of-key token. Unlike bearer tokens, the recipient of a holder-of-key token can verify if the sender of the token is legitimate.
+
+If this setting is on, the workflow is:
+
+. A token request is sent to the token endpoint in an authorization code flow or hybrid flow.
+. {project_name} requests a DPoP proof.
+. {project_name} receives the DPoP proof.
+. {project_name} successfully verifies the DPoP proof.
+
+If verification fails, {project_name} rejects the token.
+
+In the following cases, {project_name} will verify the client sending the access token or the refresh token:
+
+* A token refresh request is sent to the token endpoint with a holder-of-key refresh token.
+* A logout request is sent to Logout endpoint with a holder-of-key refresh token.
```

Comment:
```suggestion
* A logout request is sent to Logout endpoint with a holder-of-key refresh token. This is done only for public clients as described above.
```

Reply:
Sorry, the DPoP 1st step PR does not include DPoP proof and bind verification on Logout endpoint. Therefore, I would like to delete the sentence.

Reply:
Well, It's not too obvious, but I think you were correct and it actually does also for logout endpoint :-) Method `LogoutEndpoint.logoutToken()` calls `tokenManager.verifyRefreshToken`, which eventually validates DPoP binding. 

So I vote to add it back, maybe with something like:
```
* A logout request is sent to Logout endpoint with a holder-of-key refresh token. This is done only for public clients as described above.
```

What do you think?

Reply:
By the way, I think you know that this is Keycloak proprietary logout endpoint (not OIDC based) with refreshToken sent as an argument. This endpoint is used by our (deprecated) java adapters. Not sure if to emphasize it in the documentation as well?

In this case, we should possibly also update the corresponding paragraph in the MTLS HoK section above. But if you prefer to not emphasize it, it is fine to me.

Reply:
Yes, I restored the sentence for Logout Endpoint.
Also, I added "non-OIDC keycloak proprietary Logout Endpoint" to both descriptions for OAuth-MTLS and DPoP.

Reply:
Thanks!